components/x-follow-button/__tests__/x-follow-button.test.jsx

```diff
@@ -98,14 +100,14 @@ describe('x-follow-button', () => {
 				expect(subject.find('button').prop('aria-pressed')).toEqual('false')
 			})
 
-			it('button title is "Add ConceptName to myFT"', () => {
+			it('button title is "Add to myFT: ConceptName"', () => {
 				const subject = mount(<FollowButton isFollowed={false} conceptName={'ConceptName'} />)
-				expect(subject.find('button').prop('title')).toEqual('Add ConceptName to myFT')
+				expect(subject.find('button').prop('title')).toEqual('Add to myFT: ConceptName')
 			})
 
-			it('button aria-label is "Add ConceptName to myFT"', () => {
+			it('button aria-label is "Add to myFT: ConceptName"', () => {
 				const subject = mount(<FollowButton isFollowed={false} conceptName={'ConceptName'} />)
-				expect(subject.find('button').prop('aria-label')).toEqual('Add ConceptName to myFT')
+				expect(subject.find('button').prop('aria-label')).toEqual('Add to myFT: ConceptName')
```

Comment:
**thought (non-blocking)**: maybe it would be worthwhile adding some additional tests to make sure the button's text is contained in the `aria-label` property string so that we don't inadvertently create the same issue with future changes? E.g., check to make sure that this `aria-label` contains the button text, `Add to myFT` in this case.

Reply:
@ivomurrell thank you for your comment. I have implemented 2 tests that should have this function, as I believe it is a good idea. Please let me know if you think its ok :)